src/Chapter2.hs

```diff
@@ -741,7 +752,11 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]]
+contains a x
+    | null x = []
+    | a `elem` ((take 1 x) !! 0) = ((take 1 x) !! 0) : contains a (drop 1 x)
+    | otherwise = contains a (drop 1 x)
```

Comment:
That's the correct implementation :+1: But it can be simplified by using the `filter` function. You're correctly using `elem` to check if the element is in the list. Now, you can combine it with the `filter` that already keeps only those elements satisfying the given boolean function :slightly_smiling_face: